### ISSUE TITLE
Tour showing windows in wrong places on Safari for Mac when css selector is used

### ISSUE THREAD

**Issue by sestrauss** [kennibc/moodle-theme_fordson#9]
Hello,

Issue 8 shows a small tour bug on Chrome. The same tour does not really work in Safari [Version 9.1.3 (9537.86.7.8)]. The right ccs selector is highlighted. However, the description window shows up in the upper left corner. 

I have attached a zip with the theme in questions and a screen-cast of how the theme works in Safari. Note that this is the same tour I discuss in issue 8.

I have Fordson v1.2.9b 2017031400 and Moodle 3.2.1+ (Build: 20170223)

Thanks for looking into this, when you get a chance.
[Course tour in Fordson theme.zip](https://github.com/kennibc/moodle-theme_fordson/files/848409/Course.tour.in.Fordson.theme.zip)

**Comment by sestrauss** [comment]
Hello,

This seems to be the same issue as [MDL-56497](https://tracker.moodle.org/browse/MDL-56497), but in a different Mac version of Safari.

**Comment by kennibc** [comment]
I am thinking it is related to customizations you might have made or some of the plugins in use such as accessibility.  I cannot replicate the issue.  Here is a video of me using your tour on my dev site.
[user tour dev site.zip](https://github.com/kennibc/moodle-theme_fordson/files/848570/user.tour.dev.site.zip)

**Comment by sestrauss** [comment]
Thanks. My customizations are just plug-ins, but I can unistall them and see if it makes a difference.

Would you mind uploading your screencast again? I get the error "The file may be damaged or may not be a movie file that is compatible with QuickTime Player" when I try to play it.

**Comment by sestrauss** [comment]
Thanks. My customizations are just plugins, but I can try without them. Before I uninstall the blocks, can you try again with this attached tour? I may have given you the wrong tour, before. It looks different than the one I was trying to share. 

Also, your screen-cast shows Chrome. I do have an issue with Chrome, but it is different. See issue #8 for details.


[tour_export_3_1489687298.json.ics.zip](https://github.com/kennibc/moodle-theme_fordson/files/848628/tour_export_3_1489687298.json.ics.zip)

**Comment by kennibc** [comment]
Its an MP4 video with no audio.  Please use something other than quicktime.  VLC media player is great.

**Comment by sestrauss** [comment]
Hi Ken,

You are too quick! I realized that VLC worked shortly after my posting about QT and deleted that comment. After watching the video, I posted a different comment on 3/16/17 @ 2:08 PM EDT. See it above.

**Comment by kennibc** [comment]
Closing this issue since there are two issues for the same problem. The fact this is a known issue in Moodle tracker means it is not related to Fordson as it is an issue with Core.  However, look at your other issue and you will see I cannot reproduce the text running out of the box.  It may have something to do with how you are grabbing your CSS selectors.  I changed the one causing the problem and it is fixed on my end.
